omero/autogen_docs

```diff
@@ -32,7 +32,7 @@ echo "Generating DB script example"
 
 echo "Generating Web configuration templates"
 # Nginx / WSGI
-$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
```

Comment:
Looks fine. Is this also needed for the line below  too `omero web config nginx-location`?

Reply:
it is not yet directly used in the doc but it will make sense since it is under the "install-web" folder. 
This folder is used for web installation separately from OMERO.server